### PR TITLE
Add cloudbuild.yaml

### PR DESCRIPTION
**Pull request by grodowski**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    docker build --build-arg crystal_deb=crystal_0.27.0-1_amd64.deb -t "gcr.io/$PROJECT_ID/trusty_crystal:$COMMIT_SHA" .
+    if [ "$BRANCH_NAME" == "master" ]; then
+      docker tag "gcr.io/$PROJECT_ID/trusty_crystal:$COMMIT_SHA" gcr.io/$PROJECT_ID/trusty_crystal:latest
```
Comment (jbarber) [inline cloudbuild.yaml]: Missing quotes around the last argument:
```suggestion
      docker tag "gcr.io/$PROJECT_ID/trusty_crystal:$COMMIT_SHA" "gcr.io/$PROJECT_ID/trusty_crystal:latest"
```

File: cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    docker build --build-arg crystal_deb=crystal_0.27.0-1_amd64.deb -t "gcr.io/$PROJECT_ID/trusty_crystal:$COMMIT_SHA" .
+    if [ "$BRANCH_NAME" == "master" ]; then
```
Comment (jbarber) [inline cloudbuild.yaml]: If you use the `[[` operator, you don't need to quote the arguments. From TFM:
> Word splitting and pathname expansion are  not  performed on  the words between the [[ and ]]; tilde expansion, parameter and variable expansion, arithmetic expansion, command substitution, process substitution, and quote removal are performed.

Ah, shell.

File: cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      docker tag "gcr.io/$PROJECT_ID/trusty_crystal:$COMMIT_SHA" gcr.io/$PROJECT_ID/trusty_crystal:latest
+    fi
+images:
+- 'gcr.io/$PROJECT_ID/trusty_crystal'
```
Comment (jbarber) [inline cloudbuild.yaml:12]: Do single quotes here permit variable expansion? I've no idea what the variable expansion rules are for this type of file.

**Review by jbarber — APPROVED** [review]

File: cloudbuild.yaml (hunk 3 above)
Comment (grodowski) [reply cloudbuild.yaml:12]: I'd say yes, as this whole file is copy-pasted from `helm-deploy`

File: cloudbuild.yaml (hunk 2 above)
Comment (grodowski) [reply cloudbuild.yaml]: Nice 👍

**Comment by grodowski** [comment]
Cancelling - we're going to stick with `mrgrodo/trusty_crystal`. https://rainforest.slack.com/archives/C0286H6DR/p1558956089099000

**Pull request closed without merging**